src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: (number) => {
+      calculator.result += number;
+    },
```

Comment:
add empty line between methods. Please improve your code style according this comment not only in these line but on whole your solution
```suggestion
    },
    
```